recipe/meta.yaml

```diff
@@ -1,6 +1,6 @@
-{% set version = "1.4.0rc1" %}
+{% set version = "1.4.0" %}
 {% set org = "cyclus" %}
-{% set branch = "develop" %}
+{% set branch = "master" %}
 {% set sha256 = "7a6bd1334f3fd13ddac73fbb7a5218b958b793b81bda26552e9c3abbf255a905" %}
```

Comment:
I don't think of tarballs having SHAs.  Do you mean the commit SHA for the version that generates the tarball?  I guess I'm not sure how this works...  I can't seem to find documentation either.